convert2rhel/checks.py

```diff
@@ -74,13 +77,14 @@ def check_convert2rhel_latest():
     )
 
     repo_dir = os.path.join(utils.TMP_DIR, "convert2rhel_repo")
+    utils.mkdir_p(repo_dir)
     # repo_dir = os.path.join("/tmp/","convert2rhel_repo")
     repo_path = os.path.join(repo_dir, "convert2rhel.repo")
     store_content_to_file(filename=repo_path, content=repo_content)
 
     # repoquery --setopt=reposdir=$(pwd)/test_repo --releasever=8 convert2rhel
     raw_output_convert2rhel_versions, _ = run_subprocess(
-        ("repoquery --releasever={releasever} --setopt=reposdir={repodir} convert2rhel").format(
+        "repoquery --releasever={releasever} --setopt=reposdir={repodir} convert2rhel".format(
```

Comment:
Just a friendly reminder to not forget to rebase this PR later and get the latest changes from `main`, which includes now the work done in #389

Reply:
I have ported the changes that where need to follow the guidelines that where set in the PR #389